src/main/java/uk/co/ramp/covid/simulation/NetworkGenerator.java

```diff
@@ -0,0 +1,51 @@
+package uk.co.ramp.covid.simulation;
```

Comment:
Is it worth creating a new package 'output' and putting this and other output classes (DailyStats, RStats) into it?

Reply:
That sounds like a good idea, but my preference would be to merge this now, and add that to my todo list for a future PR.  (Or someone else's todo list, if anyone has time today.)

Reply:
ok